Source/gs/GSH_OpenGL/GSH_OpenGL.cpp

```diff
@@ -2071,6 +2080,39 @@ void CGSH_OpenGL::ProcessLocalToLocalTransfer()
 
 		glBindFramebuffer(GL_READ_FRAMEBUFFER, 0);
 	}
+	else if(foundSrc && !foundDest)
+	{
+		auto trxPos = make_convertible<TRXPOS>(m_nReg[GS_REG_TRXPOS]);
```

Comment:
We need a call to `FlushVertexBuffer` here to make sure all rendering commands are complete before we read from the framebuffer.

Reply:
Fixed